### PR TITLE
Ch 2: Add more swirl modules

### LINKED ISSUE #25: Ch 2: Add more swirl modules
Add `swirl` modules 6-9, 12 for use in September 2 class/homework

### PR DESCRIPTION
**Pull request by wendtke**
Closes #25

### CODE REVIEW

**Comment by wendtke** [comment]
@SmogDr - Do we need to merge this before tomorrow's class, or will the students barely start `swirl`?

**Comment by SmogDr** [comment]
They will only do the first set of modules but we should merge as soon as feasible.

**Comment by wendtke** [comment]
I'll merge after your code review/approval.

**Review by SmogDr — APPROVED** [review]

**Pull request merged**
